tdrs-backend/tdpservice/parsers/transforms.py

```diff
@@ -32,3 +32,9 @@ def ssp_ssn_decryption_func(value, **kwargs):
         decryption_table = str.maketrans(decryption_dict)
         return value.translate(decryption_table)
     return value
+
+def zero_pad(digits):
```

Comment:
is there a way to add unit tests for the transform functions? that would negate the need for a lot of the integration tests added to `test_parse.py` imo